### ISSUE TITLE
Inconsistent naming convention used in the UG and the command line program

### ISSUE THREAD

**Issue by annoy-o-mus** [annoy-o-mus/pe#10]
![image.png](https://raw.githubusercontent.com/annoy-o-mus/pe/main/files/bf59bb90-d6d7-4390-a7f7-550913a353ea.png)

![image.png](https://raw.githubusercontent.com/annoy-o-mus/pe/main/files/40da0095-1b4f-4405-bc11-6837f07cc549.png)

A few examples. Conventions also says that square brackets are optional and angled brackets are compulsory, so in that sense it makes it hard to follow

**Comment by nus-pe-script** [comment]
Your response not required for this bug as the team has accepted the bug as it is.
